src/components/application_manager/src/application_impl.cc

```diff
@@ -246,9 +247,9 @@ bool ApplicationImpl::is_voice_communication_supported() const {
   return is_voice_communication_application_;
 }
 
-void ApplicationImpl::set_voice_communication_supported(
-    bool is_voice_communication_supported) {
-  is_voice_communication_application_ = is_voice_communication_supported;
+void ApplicationImpl::set_voice_communication_supported(bool option) {
+  LOG4CXX_TRACE(logger_, "option " << option);
```

Comment:
@LuxoftAKutsan Maybe it makes sense to add std::boolalpha into the output stream for more readable log format?

Reply:
Enhancement done 35c19b0360